dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamRestControllerIT.java

```diff
@@ -1235,4 +1236,51 @@ public void closeInputStreamsDownloadWithCoverPage() throws Exception {
         Mockito.verify(inputStreamSpy, times(1)).close();
     }
 
+
+    @Test
+    public void checkContentDispositionOfFormats() throws Exception {
+        context.turnOffAuthorisationSystem();
```

Comment:
As mentioned above, I'd recommend moving the custom configuration for `webui.content_disposition_format = text/richtext, text/xml, txt` to this test method itself.  

You should be able to easily do that as follows:
```
configurationService.setProperty("webui.content_disposition_format", new String[] {
            "text/richtext",
            "text/xml",
            "txt"
        });
```

Configurations are reloaded after every test runs, so this should be all you need... the `webui.content_disposition_format` setting will go back to its default value for the next test.